### DISCUSSION TITLE
loadDynamicClasses  loadBasicClasses loadNecessaryClasses区别是什么？

### DISCUSSION THREAD

**Discussion by chuli000** [tabby-sec/tabby#23] [General]
loadDynamicClasses  loadBasicClasses loadNecessaryClasses区别是什么？

**Comment by wh1t3p1g** [comment]
loadDynamicClasses 是soot载入需要分析的class
loadBasicClasses 是为了处理soot报错的情况，这个之前应该在readme里面有所提及
loadNecessaryClasses 是为了加快分析速度，先从本地db文件载入jdk相关的cache到内存，加快分析。
